src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -18,20 +18,55 @@ import {
 
 import { CSP_IDS } from '~/platform/user/authentication/constants';
 import { signInServiceName } from '~/platform/user/authentication/selectors';
-import { isLOA1 as isLOA1Selector } from '~/platform/user/selectors';
+import { selectAuthStatus } from '../../selectors/auth-status';
 import {
-  notFoundComponent,
-  unavailableComponent,
   downloadErrorComponent,
   errorTypes,
+  notFoundComponent,
+  systemErrorComponent,
+  unavailableComponent,
 } from './utils';
 import '../../sass/download-1095b.scss';
 
-export const App = ({ displayToggle, isLOA1, loggedIn, toggleLoginModal }) => {
+export const App = ({ displayToggle, toggleLoginModal }) => {
   const [year, updateYear] = useState(0);
```

Comment:
Method signature is changed because we're now using our own selector to gather some of these values rather than mapping state to props.